### PR TITLE
Private Node for Read Requests

### LINKED ISSUE #200: Route read-only calls to Snowball private node
Snowball deployed a load balancer with private RPC Nodes connected to Avalanche Blockchain, we should use this nodes for the readonly calls so we can avoid the problems of using a public RPC like high traffic.

### PR DESCRIPTION
**Pull request by ghost**
Closes #200

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -2,3 +2,4 @@ VERSION='1.0-rc'
 APPNAME='Snowball'
 
 NETWORK=mainnet
+PRIVATENODE='https://node.snowapi.net'
```
Comment (saito-sv) [inline .env:5]: we prob want to not show this in public, can we add it to our vercel env vars instead? or use `.env` file

Comment (saito-sv) [reply .env:5]: I didnt realize this is the .env file hehe, we should ad this file to .ignore

Comment (ghost) [reply .env:5]: well, it's going to show for the users in the console anyway, it's not a big deal I think

**Review by saito-sv — APPROVED** [review]
LGTM

**Pull request merged**